src/style.css

```diff
@@ -0,0 +1,62 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  padding: 0;
+}
+
+.form {
+  box-sizing: border-box;
+  margin-top: 20px;
+  width: 100%;
+
+}
+
+.form__big {
+  height: 70px;
+  width: 100%;
+  font-size: 1em;
+  font-family: inherit;
+  padding-left: 65px;
+  line-height: 22px;
+  border-radius: 4px;
+  border: solid 1px #d7dee4;
+  background: url(images/Search.svg) border-box no-repeat 26px 24px;
+}
+
+.form__small {
+  width: 100%;
+  height: 40px;
+  padding-left: 33px;
+  line-height: 19px;
+  background: url(images/Search.svg) border-box no-repeat 14px 15px;
+  background-size: 10px;
+  border-radius: 4px;
+  border: solid 1px #d7dee4;
+}
```

Comment:
Check point 2 from checklist: you should create base class for input and all similar styles (width, border-radius, font-family, border) add for this class